packages/grid/src/GridMetricCalculator.ts

```diff
@@ -1368,11 +1383,11 @@ class GridMetricCalculator {
   }
 
   /**
-   * Get the size of the specified item
+   * Get the size (width or height) of the specified item
    * @param modelIndex The model index to get the size for
    * @param userSizes The user set sizes
    * @param calculateSize Method to calculate the size for this item
-   * @returns The size of the specified item
+   * @returns The size (width or height) of the specified item
```

Comment:
I'm still confused a bit on this

Is index a row or column model index? How does it differentiate between row 1 and column 1?

Is `userSizes` actually `rowSizes` or `columnSizes` and the index passed in corresponds to the size map you passed in?

Reply:
Changed it to say "from the size map"